### PR TITLE
Clarify behavior of sticky sessions and service weights

### PR DESCRIPTION
**Pull request by jboxman**
- https://github.com/openshift/openshift-docs/issues/15457

### CODE REVIEW

**Comment by jboxman** [comment]
@briantward, PTAL. Thanks.

@zhaozhanqi, can you confirm this is correct for 3.10, 3.9, and 3.7?

This is for https://github.com/openshift/openshift-docs/issues/15457.

**Comment by zhaozhanqi** [comment]
I'm not sure this due to bad memory. @lihongan could you help confirm this? thanks

**Comment by lihongan** [comment]
Here is the bug https://bugzilla.redhat.com/show_bug.cgi?id=1618563 but just has been fixed in 3.11 so we need update doc for 3.10, 3.9 and 3.7.

File: architecture/topics/sticky_sessions.adoc [hunk 1]
```diff
@@ -22,15 +20,17 @@ Cluster administrators can turn off stickiness for passthrough routes separately
 from other connections, or turn off stickiness entirely.
 
 By default, sticky sessions for passthrough routes are implemented using the
-`source` xref:load-balancing[load balancing strategy]. The default can be
-changed for all passthrough routes by using the `ROUTER_TCP_BALANCE_SCHEME`
-xref:env-variables[environment variable], and for individual routes by using the
+`source` xref:#load-balancing[load balancing strategy]. However, the
+`round-robin` xref:#load-balancing[load-balancing strategy] is the default when
```
Comment (lihongan) [inline architecture/topics/sticky_sessions.adoc]: Should be `roundrobin` instead of `round-robin`.

**Comment by jboxman** [comment]
@openshift/team-documentation, can someone review this? Thank you!

**Review by bergerhoffer — COMMENTED** [review]
Just one minor thing, but lgtm.

File: architecture/topics/sticky_sessions.adoc [hunk 2]
```diff
@@ -22,15 +20,17 @@ Cluster administrators can turn off stickiness for passthrough routes separately
 from other connections, or turn off stickiness entirely.
 
 By default, sticky sessions for passthrough routes are implemented using the
-`source` xref:load-balancing[load balancing strategy]. The default can be
-changed for all passthrough routes by using the `ROUTER_TCP_BALANCE_SCHEME`
-xref:env-variables[environment variable], and for individual routes by using the
+`source` xref:#load-balancing[load balancing strategy]. However, the
```
[the remark this answers is not in the document]
Comment (jboxman) [reply architecture/topics/sticky_sessions.adoc]: @bergerhoffer thanks!

File: architecture/topics/sticky_sessions.adoc (hunk 1 above)
Comment (jboxman) [reply architecture/topics/sticky_sessions.adoc]: @lihongan thanks!

**Pull request merged**